lib/matplotlib/patches.py

```diff
@@ -473,6 +473,8 @@ def get_fill(self):
     def set_capstyle(self, s):
         """
         Set the `.CapStyle`.
+        The default capstyle is ``'round'`` for `FancyArrowPatch` and
+        ``'butt'`` for all other patches.
```

Comment:
```suggestion

        The default capstyle is 'round' for `.FancyArrowPatch` and
        'butt' for all other patches.
```

https://matplotlib.org/devdocs/devel/documenting_mpl.html#quotes-for-strings
https://matplotlib.org/devdocs/devel/documenting_mpl.html#referring-to-other-code - The leading dot can be omitted if the class is in the same module. However, that can still cause problems if there are derived classes in other modules that inherit that docstring, so it's always advisable to keep the leading dot.

Reply:
changed as suggested, but see for instance
https://github.com/matplotlib/matplotlib/blob/8d7a2b9d2a38f01ee0d6802dd4f9e98aec812322/lib/matplotlib/patches.py#L4299-L4300
where the default style string is also formatted as code and without the leading dot.